rpcs3/Emu/Cell/SPUThread.cpp

```diff
@@ -6193,7 +6193,7 @@ spu_exec_object spu_thread::capture_memory_as_elf(std::span<spu_memory_segment_d
 
 		if (auto [vm_addr, ok] = vm::try_get_addr(seg.src_addr); ok)
 		{
-			if (!vm::try_access(vm_addr, data.data(), data.size(), false))
```

Comment:
You can replace this with SPU_LS_SIZE and it will be the same

Reply:
I used seg.segment_size, which is more understandable in this scope.